autosklearn/pipeline/components/data_preprocessing/feature_reduction/truncated_svd.py

```diff
@@ -70,7 +70,7 @@ def get_properties(dataset_properties: Optional[DATASET_PROPERTIES_TYPE] = None
     def get_hyperparameter_search_space(dataset_properties: Optional[DATASET_PROPERTIES_TYPE] = None
                                         ) -> ConfigurationSpace:
         cs = ConfigurationSpace()
-        # ToDo think about reasonalbe range for n_components
+        # Todo introduce a HP [0.1,...,1.0]
```

Comment:
This comment can be dropped - it's not an immediate bug and we should rather discuss this offline.